docs/user/00-10-overview-api-gateway-controller.md

```diff
@@ -8,11 +8,11 @@ APIGateway Controller is part of Kyma API Gateway Operator. Its role is to manag
 - Managing Certificates and DNS entries
 - Configuring Istio Gateways
 
-## APIGateway CR
+## APIGateway Custom Resource
 
 The `apigateways.operator.kyma-project.io` CustomResourceDefinition (CRD) describes the APIGateway CR that is used to manage the API Gateway resources. To learn more, read the [APIGateway CR documentation](./custom-resources/apigateway/04-00-apigateway-custom-resource.md).
```

Comment:
I shouldn't have changed the heading because the abbreviation is introduced [here](https://github.com/kyma-project/api-gateway/pull/776/files#diff-19e1440bfe82e2113e5a58170f0575678f71582746cfef2ae5a648b0dadc5b25R5). I'll revert my change